<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-7T1JQ3H" name="Default test" tests="15" failures="2" timestamp="22 Jul 2019 08:33:42 GMT" time="1.152" errors="0">
  <testcase name="Login" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- Login -->
  <testcase name="AddWorkgroupFromPlusIcon" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- AddWorkgroupFromPlusIcon -->
  <testcase name="EditWorkgroup" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- EditWorkgroup -->
  <testcase name="ConnectIcon" time="0.027" classname="ApodGUI.WorkgroupServer">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ApodGUI.WorkgroupServer.ConnectIcon(WorkgroupServer.java:220)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- ConnectIcon -->
  <testcase name="AddNode" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- AddNode -->
  <testcase name="DeleteWorkgroup" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- DeleteWorkgroup -->
  <testcase name="AddRemoteQueueManager" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- AddRemoteQueueManager -->
  <testcase name="ModifyRemoteQueueManager" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- ModifyRemoteQueueManager -->
  <testcase name="DeleteRemoteQueueManager" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- DeleteRemoteQueueManager -->
  <testcase name="AddRemoteEMSManager" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- AddRemoteEMSManager -->
  <testcase name="ModifyRemoteEMSServer" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- ModifyRemoteEMSServer -->
  <testcase name="DeleteRemoteEMSServer" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- DeleteRemoteEMSServer -->
  <testcase name="AddToFavoriteviewlet" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- AddToFavoriteviewlet -->
  <testcase name="SearchFilter" time="0.0" classname="ApodGUI.WorkgroupServer">
    <skipped/>
  </testcase> <!-- SearchFilter -->
  <testcase name="Logout" time="1.002" classname="ApodGUI.WorkgroupServer">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at ApodGUI.WorkgroupServer.Logout(WorkgroupServer.java:750)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Logout -->
</testsuite> <!-- Default test -->
